### PR TITLE
Meeting template modifications: October

### PR DESCRIPTION
**Pull request by patcon**
These are the meeting template modifications that @patcon is offering for consideration after his month of cultivating.

This is a work-in-progress. Comments welcome, but agenda will def be cleaned up before soliciting review.

### CODE REVIEW

File: -meeting-template.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 <sup>[from template][template] | [:notebook: Meetings][meetings] | [:date: Calendar][calendar] | [:white_check_mark: Tasks][tasks] | [:cat: Code][gh] | [:open_file_folder: Drive][gdrive]</sup>
 
-Time:      ...  
-Location:  https://meet.jit.si/hyphacoop  
-Attending: ...  
+Time:       ...  
+Location:   https://link.hypha.coop/calls
```
Comment (patcon) [inline -meeting-template.md]: Hey @ASoTNetworks, would it be cool to merge the shortlink change in your pending PR? A feel a shortlink will make this transition less confusing, and I've been waiting to let you handle it, but I'm not sure the delay

Comment (dcwalk) [reply -meeting-template.md]: Just chiming in to say this is one case where I think pointing directly to our jitsi is important -- that is the link that has to be pasted into the app to join a call, so hiding it actually makes it difficult for folks joining on mobile.

**Comment by dcwalk** [comment]
@patcon what is status of this? Are we working from this template? I'd like to get a handle before taking over cultivating next week.

File: -meeting-template.md (hunk 1 above)
Comment (patcon) [reply -meeting-template.md]: gonna resolve that concern this weekend, I think: https://github.com/jitsi/jitsi-meet/issues/4801 😉 

Any other concerns with using it? I'm easy

Comment (benhylau) [reply -meeting-template.md]: Does `link.hypha.coop/calls` actually make sense when hypha can have calls in other rooms on our jitsi server?

Comment (patcon) [reply -meeting-template.md]: The point, in my understanding, is to have our current main room linked in docs/events/etc that we never have to update, regardless of our platform (for example, if jitsi goes down one morning, or we move platforms next year).

So it going going to the root url doesn't actually address that...?

Comment (patcon) [reply -meeting-template.md]: Not precious about /calls being the link. Fine to send /calls to root, and make another direct to our main room for docs

**Comment by patcon** [comment]
@dcwalk thanks for poke. If it looks ok, I'll merge over weekend, and create a stub for your next call (as is a #todo for cultivator closing out meeting)

File: -meeting-template.md (hunk 1 above)
Comment (benhylau) [reply -meeting-template.md]: I understand the motivation of this, I followed the discussion where this came about, but I am raising that `link.hypha.coop/calls` is misleading and in this case I'd rather be explicit about `jitsi.hypha.coop/hyphacoop`. There are many reasons, like what dc raised about redirect, or wanting to look back at notes and see on what platform a call took place, etc. The work to change up a link in a template is not that high, we don't really need a shortlink for it.

Comment (patcon) [reply -meeting-template.md]: - :heart: I feel like I want to look back on notes and know platform (ben)
- :heart: I feel a short link to our main room is misleading. (Ben)
- :heart: I feel change up links in template docs is not high (ben)
  - :mag: it has taken us two weeks to update links to pilot (patcon)

Is the above a fair summary?

Comment (patcon) [reply -meeting-template.md]: - :heart: just bc we have our own jitsi now with infinite rooms, that does not mean we need to care about or plan to use them, or complicate our ways of working

Comment (patcon) [reply -meeting-template.md]: - :bulb: call it link.hypha.coop/partyline or link.hypha.coop/mainline, and send link.hypha.coop/calls to root

Comment (dcwalk) [reply -meeting-template.md]: Okay so two folks (me and @benhylau have requested we use the "bare" link for now, while the other options work through I'm going to merge with that approach, b/c our next meeting is, gulp, today. 

We can revise in a future template

Comment (dcwalk) [inline -meeting-template.md]:
```suggestion
Location:   https://jitsi.hypha.coop/hyphacoop  
```

**Comment by dcwalk** [comment]
Merging to unblock my cultivating in November

**Pull request merged**